Makefile.am

```diff
@@ -19,7 +19,6 @@ istep0_LDADD = libipl.la
 ipl_SOURCES = ipl.C
 ipl_LDADD = libipl.la
 ipl_LDFLAGS = -Wl,--whole-archive,-L.libs,-lipl,-lpdbg,--no-whole-archive
-
```

Comment:
Looks you deleted one line? Maybe by mistake?

Reply:
agree